ansible/roles/lvm-partition/tasks/main.yml

```diff
@@ -3,21 +3,22 @@
 
 - name: storage | create logical volume
   lvol:
-    vg: "{{ vgname }}"
-    lv: "{{ lvname }}"
-    size: "{{ lvsize }}"
-    opts: "{{ lvopts }}"
+    vg: "{{ lvm_vgname }}"
+    lv: "{{ lvm_lvname }}"
+    size: "{{ lvm_lvsize }}"
+    opts: "{{ lvm_lvopts }}"
 
 - name: storage | format
   filesystem:
-    fstype: "{{ lvfilesystem }}"
-    dev: /dev/{{ vgname }}/{{ lvname }}
+    fstype: "{{ lvm_lvfilesystem }}"
+    dev: /dev/{{ lvm_vgname }}/{{ lvm_lvname }}
+    resizefs: yes
 
 - name: storage | mount scratch
```

Comment:
Since you're cleaning up this file, can you change the name to `storage | mount`?